docs/system/hardware-metrics.md

```diff
@@ -44,43 +42,8 @@ when creating instruments not explicitly defined in the specification.
 > * SHOULD introduce a control mechanism to allow users to opt-in to the new
 >   conventions once the migration plan is finalized.
 
-## Common hardware attributes
-
-All metrics in `hw.` instruments should be attached to a [Host Resource](/docs/resource/host.md)
-and therefore inherit its attributes, like `host.id` and `host.name`.
-
-Additionally, all metrics in `hw.` instruments have the following attributes:
-
-| Attribute Key | Description                                                                                                   | Example                             | Requirement Level |
-| ------------- | ------------------------------------------------------------------------------------------------------------- | ----------------------------------- | ----------------- |
-| `id`          | An identifier for the hardware component, unique within the monitored host                                    | `win32battery_battery_testsysa33_1` | **Required**      |
-| `name`        | An easily-recognizable name for the hardware component                                                        | `eth0`                              | Recommended       |
-| `parent`      | Unique identifier of the parent component (typically the `id` attribute of the enclosure, or disk controller) | `dellStorage_perc_0`                | Recommended       |
-
 ## Metric Instruments
 
-### `hw.` - Common hardware metrics
```

Comment:
Can we keep HW metrics in the same file? 
If we're splitting them in 2, we should at least have cross-references and some explanation why they are different

/cc @open-telemetry/semconv-system-approvers

Reply:
Sorry, I don't understand your question? In which file?

Reply:
Currently hardware metrics are spread across multiple files. I understand the intention to move all of them, but we should keep things consistent in the mean time.

Since you're removing some metrics from this file, please at least add a link to the new doc from this one.

Reply:
Please add a link to the common metrics section of the new document, so people would see it among other metrics (and not just in the note above)

Reply:
Updated